gatsby-config.js

```diff
@@ -83,6 +83,31 @@ module.exports = {
     },
     'gatsby-plugin-catch-links',
     'gatsby-plugin-sitemap',
+    {
+      resolve: 'gatsby-plugin-robots-txt',
+      options: {
+        host: config.siteUrl + pathPrefix,
+        sitemap: config.siteUrl + pathPrefix + 'sitemap.xml',
+        env: {
+          development: {
+            policy: [{ userAgent: '*', disallow: ['/'] }],
+          },
```

Comment:
```suggestion
```
@dannyrb I guess this part is irrevelevant. No external build is going to have `development` in `NODE_ENV`. What do u think?